Makefile

```diff
@@ -181,6 +185,7 @@ jupyter-notebook: base
 capella/base: SHELL=./capella_loop.sh
 capella/base: base
 	envsubst < capella/.dockerignore.template > capella/.dockerignore
+	cp eclipse/set_memory_flags.py capella/startup/set_memory_flags.py
```

Comment:
> What is the directory capella/startup for? 

All executable shell and Python scripts in this directory will be executed during container startup.

> Does it make sense to rename the dir from startup to setup and put a README.md into that dir so that people can easily read what it is about?

For consistency that's a good idea. I'll rename startup to setup. 
I'll also add a small README to the directory for better transparency.